### ISSUE TITLE
Search for key address

### ISSUE THREAD

**Issue by traumschule** [polkadot-js/extension#1171]
This is a feature request to search for accounts by address.

Currently only labels are queried. 

Also expanding the search field by default would suit better (setting).

**Comment by traumschule** [comment]
Someone add the label please.

**Comment by traumschule** [comment]
Sometimes it takes more than 21 days until a developer finds time.

**Comment by Tbaut** [comment]
>  to search for accounts by address.

agreed that this would be a welcome addition, I'll make a PR. I disagree regarding the search open by default. Also this will conflict with the connected accounts that will be available in the next release. See https://blog.chainsafe.io/polkadot-js-extension-new-privacy-feature-3f80900589ec
